module/actors/actor.js

```diff
@@ -94,12 +94,8 @@ export class CoCActor extends Actor {
     if (['character', 'npc', 'creature'].includes(this.type)) {
       this.system.skills = {}
       for (const i of this.items) {
-        if (i.type === 'skill') {
-          this.system.skills[`${i.system.skillName}`] = {
-            value: i.rawValue
-          }
-          this.system.skills[`${i.id}`] = { value: i.rawValue }
-        }
+        if (i.type !== 'skill') continue
+        this.system.skills[`${i.itemIdentifier}`] = { value: i.rawValue }
```

Comment:
My worry is that I didn't found all of the usages where `itemIdentifier` should be used (both getters and setts) across the module...